kpi/views.py

```diff
@@ -907,6 +1108,18 @@ def perform_create(self, serializer):
             user = get_anonymous_user()
         serializer.save(owner=user)
 
+    def partial_update(self, request, *args, **kwargs):
+        instance = self.get_object()
+
+        if CLONE_ARG_NAME in request.data:
+            serializer= self._get_clone_serializer(instance)
```

Comment:
whitespace 🤣 sorry, couldn't resist!

Reply:
Fair enough :-D. I can't believe I missed that one. Bad copy&paste :P